### PR TITLE
Commerce 6189 URLs added to Admin Catalog API Products get response

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-6189

This PR introduce the URL for Products in the headless-commerce-admin-catalog-impl API.

Schermata 2021-03-23 alle 11 08 55

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java [hunk 1]
```diff
@@ -166,6 +172,23 @@ private String _getSku(CPDefinition cpDefinition, Locale locale) {
...
+
+		Set<Map.Entry<Locale, String>> entries = urlTitleMap.entrySet();
+
+		Stream<Map.Entry<Locale, String>> stream = entries.stream();
+
+		return stream.collect(
+			Collectors.toMap(
+				e -> {
+					Locale key = e.getKey();
+
```
Comment (dsitu) [inline modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-catalog-impl/src/main/java/com/liferay/headless/commerce/admin/catalog/internal/dto/v1_0/converter/ProductDTOConverter.java:186]: We should name this entry.
```suggestion
				entry -> {
					Locale key = entry.getKey();

```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Review by dsitu — APPROVED** [review]

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:reevaluate:1047401_7875

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward

**Pull request closed without merging**